src/seed_data/utils/utilFunctions.ts

```diff
@@ -0,0 +1,51 @@
+import * as faker from 'faker';
+import { LetterGrade } from '../../entity/Enums';
+
+export function getRandomIntInclusive(min: number, max: number) {
```

Comment:
correct me if im wrong, but i think(?) this is only being used once in `Course.factory.ts` and it seems pretty similar to what `faker.random.number({ min: num, max: num)}` is doing